packages/blocks/image/src/components/resize-image-block.tsx

```diff
@@ -1,4 +1,9 @@
-import React, { useLayoutEffect, useRef } from "react";
+import {
+  FunctionComponent,
+  MouseEvent as ReactMouseEvent,
```

Comment:
Here is a slightly nasty part of using named React imports. `MouseEvent` from React shadows DOM `MouseEvent` and we can no longer use it. I had to import `MouseEvent as ReactMouseEvent` in four files. It was `React.MouseEvent` previously.

Reply:
Nice idea! Should be fixed in https://github.com/hashintel/hash/pull/847/commits/ae2aca943adb830e0182d5baf1ec8851b76fbbab